src/index.css

```diff
@@ -13,11 +13,31 @@ a {
   text-decoration: none;
 }
 
+h1, h2, h3,h4, h5 {
```

Comment:
Be aware of styling specific tags across your app, you may encounter sometimes components being affected by global styles. Styled Components or local scope selectore are a good solution